test/uk/gov/hmrc/agentsubscriptionfrontend/models/MessagesValidationSpec.scala

```diff
@@ -0,0 +1,42 @@
+package uk.gov.hmrc.agentsubscriptionfrontend.models
+
+import uk.gov.hmrc.agentsubscriptionfrontend.models.Address._
+import org.scalatestplus.play.{OneAppPerSuite, PlaySpec}
+import play.api.data.validation.ValidationError
+
+class MessagesValidationSpec extends PlaySpec with OneAppPerSuite {
+
+  "renderErrors function" should {
+    "concatenate invalid and blacklist error messages" in {
+      val validationErrors = Set(ValidationError("error.postcode.invalid"), ValidationError("error.postcode.blacklisted"))
+      val result: String = renderErrors(validationErrors)
+
+      result mustEqual " You have entered an invalid postcode, You can't use the postcode you've entered"
```

Comment:
Ah, now I understand what the `substring(1)` in `renderErrors` is there for :)

It would be nice to avoid the leading space, I will push a commit that does this.